galleries/examples/event_handling/viewlims.py

```diff
@@ -77,17 +87,14 @@ def ax_update(self, ax):
 ax2.imshow(Z, origin='lower',
            extent=(md.x.min(), md.x.max(), md.y.min(), md.y.max()))
 
-rect = UpdatingRect(
-    [0, 0], 0, 0, facecolor='none', edgecolor='black', linewidth=1.0)
-rect.set_bounds(*ax2.viewLim.bounds)
-ax1.add_patch(rect)
-
-# Connect for changing the view limits
-ax2.callbacks.connect('xlim_changed', rect)
-ax2.callbacks.connect('ylim_changed', rect)
```

Comment:
Ah, I see this part is missing, namely that zooming on the right Axes updates the selector on the left.